core-bundle/src/Resources/contao/templates/backend/be_wildcard.html5

```diff
@@ -8,7 +8,7 @@
     <span class="upper"><?= $this->wildcard ?></span>
     <?php if ($this->link): ?>
       <br>
-      <?= $this->link ?> (<a href="<?= $this->href.'&amp;rt='.REQUEST_TOKEN ?>" class="tl_gray">ID: <?= $this->id ?></a>)
+      <?= $this->link ?> (<a href="<?= $this->href.'&amp;rt='.Contao\System::getContainer()->get('contao.csrf.token_manager')->getDefaultTokenValue() ?>" class="tl_gray">ID: <?= $this->id ?></a>)
```

Comment:
Not sure if I recall it wrong but haven‘t we talked about adding a `requestToken()` method to the `TemplateInheritance` class and/or Twig environment, so we do not have to pass `$template->requestToken` in every controller or call this method in every template?

Reply:
I think we decided that it should be the responsibility of the Controller, see https://github.com/contao/contao/pull/3620#issuecomment-961157715

Reply:
Can we discuss this again? Because a helper method would be so much more convenient IMHO.

Reply:
I fully agree with @leofeyer

Reply:
In my view, only the controller can know which tokens to pass to the templates. I thought our goal was to move away from global usage (that must also be checked globally).

Reply:
I agree that this should be the ultimate goal. However, we'll only be able to change that in a major version because everyone is relying on the fact that every POST request is validated (and to be honest, I'm not even sure that's a bad thing). If we want to change, we can still change the implementation in the controllers then too. But as long as it's a global check, a global function does seem to make sense to me too.

Reply:
Done